bower.json

```diff
@@ -51,7 +51,7 @@
     "jquery-min-1-11-3": "https://code.jquery.com/jquery-1.11.3.min.js",
     "jquery-min-2-2-0": "https://code.jquery.com/jquery-2.2.0.min.js",
     "jquery-min-3-1-1": "https://code.jquery.com/jquery-3.1.1.min.js",
-    "jquery-modern-blink-0-1-3":"https://github.com/leonderijke/jQuery-Modern-Blink.git#0.1.3",
+    "jquery-modern-blink-0-1-3": "https://github.com/leonderijke/jQuery-Modern-Blink.git#0.1.3",
```

Comment:
Seems like Bower did this automatically when I ran `bower install bootstrap-sidebar --save` Just a whitespace fix